metrics/client.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	RPCCallLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name: "rpc_call_latency",
+		Help: "The duration of an RPC call in nanoseconds",
+		Buckets: []float64{
+			float64(50 * time.Millisecond),
+			float64(100 * time.Millisecond),
+			float64(200 * time.Millisecond),
+			float64(500 * time.Millisecond),
+			float64(1 * time.Second),
+			float64(2 * time.Second),
+			float64(4 * time.Second),
+			float64(8 * time.Second),
+		},
+	}, []string{"ChainFamily", "ChainID", "rpcUrl", "isSendOnly", "success", "rpcCallName"})
```

Comment:
I think we want all labels to be camelCased (e.g. `chainFamily`, `chainID`) and aligned across metrics